src/frontend/src/components/Post/Post.jsx

```diff
@@ -18,9 +18,10 @@ const useStyles = makeStyles((theme) => ({
   header: {
     backgroundColor: theme.palette.primary.main,
     color: theme.palette.text.secondary,
-    padding: '1.4em',
+    padding: '2em 3em 2em 3em',
     lineHeight: '1.3',
     zIndex: 1500,
+    top: '-1.1em',
     [theme.breakpoints.down(1440)]: {
```

Comment:
I don't know when this got changed, but it's making it so that your fix gets overridden  in all screens:
![image](https://user-images.githubusercontent.com/49992710/99478417-5b296b00-2922-11eb-90db-763429b6280a.png)

Reply:
I double checked and the padding for smaller screens was still set to .7em. I changed it back to the original 2em and everything should be working fine on smaller screens now.

![image](https://user-images.githubusercontent.com/44535539/99482424-d6dae600-2929-11eb-84fe-02bb243db70c.png)